Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.yml

```diff
@@ -1347,11 +1353,135 @@ script:
+      auto: PREDEFINED
+      defaultValue: 'false'
+      predefined:
+      - 'true'
+      - 'false'
+    description: 'Updates an enrollment with changes. Response type will vary depending on the type and impact of change. For example, changing SANs list may return HTTP 202 Accepted since the operation require a new certificate and network deployment operations, and thus cannot be completed without a change. On the contrary, for example a Technical Contact name change may return HTTP 200 OK assuming there are no active change and when the operation does not require a new certificate. Reference: https://techdocs.akamai.com/cps/reference/put-enrollment  NOTES: Depending on the type of the modification, additional steps might be required to complete the update. These additional steps could be carrying out a "renew" change by resubmitting the CSR, acknowleging the warnings raised then waiting for the certificate to be deployed into PRODUCTION. However these additional steps are not included in this command. User needs to conduct those steps once the update command is completed.'
+    name: akamai-update-cps-enrollment
+    outputs:
+    - contextPath: Akamai.Enrollment.Changes
+      description: Akamai Enrollment Changes.
+      type: Dictionary
+  - arguments:
+    - description: Enrollment path found in the pending change location field.
+      name: enrollment_path
+    - description: Enrollment ID on which to perform the desired operation. And it can be retrived via list_enrollments_command.
+      name: enrollment_id
+    - description: Chnage ID on which to perform the desired operation. And it can be retrived via list_enrollments_command.
+      name: change_id
+    - description: 'The time after when the change will no longer be in effect.             This value is an ISO-8601 timestamp. (UTC)             Sample: 2021-01-31T00:00:00.000Z.'
+      name: deploy_not_after
+    - description: 'The time that you want change to take effect. If you do not set this, the change occurs immediately,             although most changes take some time to take effect even when they are immediately effective.             This value is an ISO-8601 timestamp. (UTC)             Sample: 2021-01-31T00:00:00.000Z.'
+      name: deploy_not_before
+      required: true
+    description: Updates the current deployment schedule.
+    name: akamai-update-cps-enrollment-schedule
+    outputs:
+    - contextPath: Akamai.Enrollment.Changes
+      description: Akamai Enrollment Changes.
+      type: Dictionary
+  - arguments:
+    - description: 'Enrollment path found in the pending change location field.'
+      name: enrollment_path
+    - description: The enrollment ID on which to perform the desired operation. And it can be retrived via list_enrollments_command.
+      name: enrollment_id
+    - description: The change for this enrollment on which to perform the desired operation.  And it can be retrived via list_enrollments_command.
```

Comment:
```suggestion
    - description: The change for this enrollment on which to perform the desired operation.  It can be retrieved via the akamai-list-enrollments command.
```